WordPress/Classes/ViewRelated/Site Creation/SiteSegments/SiteSegmentsWizardContent.swift

```diff
@@ -26,19 +30,91 @@ final class SiteSegmentsWizardContent: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        applyTitle()
+        setupBackground()
+        setupTable()
+        initCancelButton()
+    }
+
+    private func applyTitle() {
+        title = NSLocalizedString("Create Site", comment: "Site creation. Step 1. Screen title")
+    }
+
+    private func setupBackground() {
+        view.backgroundColor = WPStyleGuide.greyLighten30()
+    }
+
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        setupTable()
 
         fetchSegments()
     }
 
     private func setupTable() {
+        setupTableBackground()
+        setupCell()
+        setupHeader()
+        hideSeparators()
+    }
+
+    private func setupTableBackground() {
+        table.backgroundColor = WPStyleGuide.greyLighten30()
+    }
+
+    private func hideSeparators() {
+        table.tableFooterView = UIView(frame: .zero)
+    }
+
+    private func setupCell() {
+        registerCell()
+        setupCellHeight()
+    }
+
+    private func registerCell() {
         let cellName = SiteSegmentsCell.cellReuseIdentifier()
         let nib = UINib(nibName: cellName, bundle: nil)
         table.register(nib, forCellReuseIdentifier: cellName)
     }
 
+    private func setupCellHeight() {
+        table.rowHeight = StyleConstants.rowHeight
+        table.estimatedRowHeight = StyleConstants.rowHeight
+        table.separatorInset = StyleConstants.separatorInset
+    }
+
+    private func setupHeader() {
+        let header = TitleSubtitleHeader(frame: .zero)
+        header.setTitle(headerData.title)
+        header.setSubtitle(headerData.subtitle)
+
+        table.tableHeaderView = header
+
+        // This is the only way I found to insert a stack view into the header without breaking the autolayout constraints. We do something similar in Reader
+        header.centerXAnchor.constraint(equalTo: table.centerXAnchor).isActive = true
```

Comment:
Rather than setting `.isActive = true` on each constraint, might I suggest wrapping these constraint creation calls in `NSLayoutConstraint.activate(_ constraints: [NSLayoutConstraint])`? 

The [documentation]((https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activate)) suggests the following:

> Typically, using this method is more efficient than activating each constraint individually.